examples/every_option.rs

```diff
@@ -0,0 +1,97 @@
+use bevy::{prelude::*, ui::FocusPolicy, window::PrimaryWindow};
+use bevy_cosmic_edit::{
+    bevy_color_to_cosmic, change_active_editor_sprite, change_active_editor_ui, get_x_offset,
+    ActiveEditor, Attrs, AttrsOwned, CosmicAttrs, CosmicBackground, CosmicEditPlugin,
+    CosmicEditUiBundle, CosmicEditor, CosmicMaxChars, CosmicMaxLines, CosmicMetrics, CosmicText,
+    CosmicTextPosition, Edit,
+};
+
+#[derive(Resource)]
+struct TextChangeTimer(pub Timer);
+
+fn setup(mut commands: Commands, windows: Query<&Window, With<PrimaryWindow>>) {
+    commands.spawn(Camera2dBundle::default());
+
+    let attrs =
+        AttrsOwned::new(Attrs::new().color(bevy_color_to_cosmic(Color::rgb(0.27, 0.27, 0.27))));
+    let primary_window = windows.single();
+
+    let editor = commands
+        .spawn(CosmicEditUiBundle {
+            node: Node::default(),
+            button: Button,
+            visibility: Visibility::Visible,
+            computed_visibility: ComputedVisibility::default(),
+            z_index: ZIndex::default(),
+            image: UiImage::default(),
+            transform: Transform::default(),
+            interaction: Interaction::default(),
+            focus_policy: FocusPolicy::default(),
+            text_position: CosmicTextPosition::default(),
+            background_color: BackgroundColor::default(),
+            global_transform: GlobalTransform::default(),
+            background_image: CosmicBackground::default(),
+            border_color: Color::LIME_GREEN.into(),
+            style: Style {
+                // Size and position of text box
+                border: UiRect::all(Val::Px(4.)),
+                width: Val::Percent(20.),
+                height: Val::Px(50.),
+                left: Val::Percent(40.),
+                top: Val::Px(100.),
+                ..default()
+            },
+            cosmic_attrs: CosmicAttrs(attrs.clone()),
+            cosmic_metrics: CosmicMetrics {
+                font_size: 16.,
+                line_height: 16.,
+                scale_factor: primary_window.scale_factor() as f32,
+            },
+            max_chars: CosmicMaxChars(15),
+            max_lines: CosmicMaxLines(1),
+            set_text: CosmicText::OneStyle("BANANA IS THE CODEWORD!".into()),
```

Comment:
it can't be `text` right? maybe then call it `cosmic_text` for sprite and ui elements.

Reply:
it was `text` while i was working on it, no reason for it to not be, but it currently only works as a setter, it's not synced with buffer contents

Reply:
so for ui bundle it only works as setter but for sprite as setter and getter?

Reply:
For both it is only a setter, is used in `cosmic_editor_builder` and `update_buffer_text` (the latter waits for `Changed` event on the `CosmicText` and then updates the buffer). I'm in favor of calling it text and syncing it correctly later tho